admin-ui/plugins/fido/helper/utils.ts

```diff
@@ -56,14 +66,21 @@ const transformToFormValues = (configuration?: Record<string, unknown>, type?: s
         metricReporterInterval: configuration?.metricReporterInterval || '',
         metricReporterKeepDataDays: configuration?.metricReporterKeepDataDays || '',
         personCustomObjectClassList: configuration?.personCustomObjectClassList || [],
-        hints: arrayValidationWithSchema(
-          (configuration as any)?.fido2Configuration?.hints,
-          PublicKeyCredentialHints,
-        ),
+        hints: arrayValidationWithSchema(fido2Config?.hints || [], PublicKeyCredentialHints),
       }
 }
 
-const createFidoConfigPayload = ({ fidoConfiguration, data, type }: any) => {
+interface CreateFidoConfigPayloadParams {
+  fidoConfiguration: { fido: AppConfiguration }
+  data: Record<string, unknown>
+  type: string
+}
```

Comment:
Narrow types for stronger contracts

Consider restricting `type` to a union and tightening `data` with discriminated shapes for STATIC/DYNAMIC to catch mapping drift at compile time.

Example:

```ts
type ConfigKind = typeof fidoConstants.STATIC | typeof fidoConstants.DYNAMIC

interface CreateFidoConfigPayloadParams {
  fidoConfiguration: { fido: AppConfiguration }
  data: StaticFormValues | DynamicFormValues
  type: ConfigKind
}
```